app/src/main/java/com/hedvig/app/feature/crossselling/ui/CrossSellTracker.kt

```diff
@@ -27,4 +27,22 @@ class CrossSellTracker(
             }
         }
     }
+
+    fun notificationReceived(crossSell: CrossSellData?) {
```

Comment:
Can we pass only `crossSellType` here instead of `CrossSellData`?

Reply:
We can 🙂 I intentionally let these methods accept the entire `CrossSellData`-object here, as the way I saw it, the calling code should not concern themselves with which part of the `CrossSellData` should be tracked 😅